### PR TITLE
Updated models to support Polylines and LatLng

### PR DESCRIPTION
**Pull request by ooms97**
Updated coordinate and path model to support polylines and latlng for google maps. Unit tests were also updated.

### CODE REVIEW

File: lib/models/coordinate.dart [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import 'dart:collection';
```
Comment (muhanator) [inline lib/models/coordinate.dart]: delete newLine and make both imports on the same line

File: lib/models/path.dart [hunk 2]
```diff
@@ -1,3 +1,8 @@
+import 'dart:collection';
+
```
Comment (muhanator) [inline lib/models/path.dart]: ditto

File: lib/models/path.dart [hunk 3]
```diff
@@ -1,3 +1,8 @@
...
+import 'package:flutter/material.dart';
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+
```
Comment (muhanator) [inline lib/models/path.dart]: ditto

Comment (ooms97) [reply lib/models/path.dart]: done.

File: lib/models/path.dart (hunk 2 above)
Comment (ooms97) [reply lib/models/path.dart]: done.

File: lib/models/coordinate.dart (hunk 1 above)
Comment (ooms97) [reply lib/models/coordinate.dart]: done.

**Pull request merged**